apps/token-manager/app/src/App.js

```diff
@@ -57,7 +59,7 @@ class App extends React.Component {
   getHolderBalance = address => {
     const { holders } = this.props
     const holder = holders.find(holder =>
-      addressesEqual(holder.address, address)
+      addressesEqual(holder.address, address),
```

Comment:
I realize we don’t have a `.prettierrc` per app, but we should use the ES5 style trailing comma ([`trailingComma: "es5"`](https://github.com/aragon/aragon-apps/blob/master/apps/finance/app/.prettierrc#L4)) for consistency.